web/client/utils/EpicsUtils.js

```diff
@@ -35,3 +43,42 @@ const defaultEpicWrapper = (epic, k = "--unknown--") => (...args) =>
  */
 export const wrapEpics = (epics, wrapper = defaultEpicWrapper) =>
     Object.keys(epics).map(k => wrapper(epics[k], k));
+
+
+/**
+ * Common part of the workflow that toggles one plugin off when another plugin intend to perform drawing action
+ * @param action$
+ * @param store
+ * @param {string} toolName - tool name(s) that should be toggled off
+ * @param {function} apply - optional function to override action triggered by default
+ * @param isActiveCallback - optional function to override callback to check tool activeness
+ * @returns {Observable<unknown>}
+ */
+export const shutdownToolOnAnotherToolDrawing = (action$, store, toolName,
```

Comment:
This EpicUtils.js provides high level general function for the app setup and it is imported by the StateUtils and PluginsUtils, utils used in different files, while this new function is used specifically to check the controls applied to panels and draw support of the product app.

For this reason I would prefer to move this from EpicUtils.js to a new utils file that we can call ControlUtils.js to isolate this function.

Reply:
Good suggestion, applied